info.plist

```diff
@@ -225,40 +259,61 @@ end alfred_script</string>
 	<dict>
 		<key>2F2D9399-94AC-4A44-99BE-993A4A623AC2</key>
 		<dict>
+			<key>xpos</key>
+			<integer>500</integer>
 			<key>ypos</key>
 			<real>320</real>
 		</dict>
 		<key>58619DEC-D6A7-4344-B557-5617B39F12DF</key>
 		<dict>
+			<key>xpos</key>
+			<integer>500</integer>
 			<key>ypos</key>
 			<real>10</real>
 		</dict>
 		<key>6079855E-22E9-4B4C-8DD5-919CB258064A</key>
 		<dict>
+			<key>xpos</key>
+			<integer>100</integer>
 			<key>ypos</key>
 			<real>200</real>
 		</dict>
 		<key>7383993A-C301-4ABD-B9A8-1E115F8F4573</key>
 		<dict>
+			<key>xpos</key>
+			<integer>500</integer>
 			<key>ypos</key>
 			<real>200</real>
 		</dict>
 		<key>90677662-35FF-41CB-93F4-6758357932B1</key>
 		<dict>
+			<key>xpos</key>
+			<integer>300</integer>
 			<key>ypos</key>
 			<real>10</real>
 		</dict>
 		<key>B8455555-F05D-4969-83C9-5DE09914B79D</key>
 		<dict>
+			<key>xpos</key>
+			<integer>300</integer>
 			<key>ypos</key>
 			<real>120</real>
 		</dict>
 		<key>FA1D6F95-2A45-46E1-A83E-9C93D722AB02</key>
 		<dict>
+			<key>xpos</key>
+			<integer>100</integer>
 			<key>ypos</key>
 			<real>320</real>
 		</dict>
 	</dict>
+	<key>variables</key>
```

Comment:
This is and the next few lines are the relevant part in the `info.plist`. Everything else was changed automatically by Alfred and pertains to its recent features.

To make sure Alfred didn’t change anything it didn’t need to, I redownloaded and changed it’s `info.plist` with the one from this repo before making the change.